### ISSUE TITLE
Having trouble decoding audioMessage

### ISSUE THREAD

**Issue by gfarias** [adiwajshing/Baileys#56]
Hello, 

I found an error when trying to decode an audio message. I'm getting "HMAC sign does not match audioMessage"
```javascript
const [notificationType, messageType] = wa.default.getNotificationType(message);
await wa.default.decodeMediaMessage(message.message, message.key.id) 
		.then (async (filename, two) => {})
		.catch (err => {
			console.log("ERROR: " + err, messageType)
		})
```
Is anyone else running into this?

**Comment by adiwajshing** [comment]
Hi, thanks for bringing this to notice. It's been fixed in the latest release (version 2.1.0)

Do let me know if it works.

**Comment by adiwajshing** [comment]
Assuming it's fixed, closing.
